src/plugins.js

```diff
@@ -0,0 +1,131 @@
+export default {
+  // es2015
+  "transform-es2015-arrow-functions": {
+    edge: 13,
+    firefox: 45,
+    chrome: 49,
```

Comment:
I'm not sure I follow what you're saying. Is this an issue with the transformation or with the implementation in that version of node, with the data or with this specific comparator. Can you suggest something different?

I mainly intend to follow a simplified version of the data the [caniuse DB](https://github.com/Fyrd/caniuse/blob/master/CONTRIBUTING.md#supported-changes) follows. Of course we won't care much about a partial implementation since we just need to know whether or not to apply a plugin.